core/util/strings.js

```diff
@@ -9,6 +9,6 @@
 /*
  * Decode from UTF-8
  */
-export function decodeUTF8 (utf8string) {
+export function decodeUTF8(utf8string) {
```

Comment:
this commit doesn't seem to bring consistency at all!

Reply:
**Anonymous function**
function () {....} // One single space between function and opening parenthesis and between closing parenthesis and opening curly brace

**Named function**
function funcName() {....} // No space between function name and opening parenthesis and single space between closing parenthesis and opening curly brace

It's the airbnb recommended way. I can follow some other way if you would prefer that.